observer/probers/http/http_conf.go

```diff
@@ -54,10 +56,17 @@ func (c HTTPConf) validateRCodes() error {
 	return nil
 }
 
+func (c HTTPConf) validateCollectors(colls map[string]*prometheus.Collector) error {
+	if colls != nil {
+		return errors.New("http prober defines no metrics but received non-nil collector map")
+	}
+	return nil
+}
+
 // MakeProber constructs a `HTTPProbe` object from the contents of the
 // bound `HTTPConf` object. If the `HTTPConf` cannot be validated, an
 // error appropriate for end-user consumption is returned instead.
-func (c HTTPConf) MakeProber() (probers.Prober, error) {
+func (c HTTPConf) MakeProber(colls map[string]*prometheus.Collector) (probers.Prober, error) {
```

Comment:
You can use a blank identifier here.

```suggestion
func (c HTTPConf) MakeProber(_ map[string]*prometheus.Collector) (probers.Prober, error) {
```